src/main/resources/db/changelog/changelog.sql

```diff
@@ -423,3 +423,8 @@ ADD CONSTRAINT `unique_extId` UNIQUE (studyId, externalId);
 
 ALTER TABLE `AccountRoles`
 MODIFY COLUMN `role` enum('DEVELOPER','RESEARCHER','ADMIN','ORG_ADMIN','WORKER','SUPERADMIN') NOT NULL;
+
+-- changeset bridge:23
+
+ALTER TABLE `AccountRoles`
+MODIFY COLUMN `role` enum('DEVELOPER','RESEARCHER','ADMIN','ORG_ADMIN','STUDY_COORDINATOR','WORKER','SUPERADMIN') NOT NULL;
```

Comment:
Inserting a new enum value in the middle of the list can have pretty major performance implications https://stackoverflow.com/questions/9638801/changing-an-enum-type-in-mysql

It would be better to always add enum values to the end of the list.

Reply:
I didn't know this...I'll do this from now on (changed).